### ISSUE TITLE
Context menu key doesn't work for regular blobs/disks

### ISSUE THREAD

**Issue by v-xianya** [microsoft/AzureStorageExplorer#6694]
**Storage Explorer Version**: 1.29.0-dev
**Build Number**: 20230321.12
**Branch**: main
**Platform/OS**: Windows 10/Linux Ubuntu 22.04
**Architecture**: ia32/x64
**How Found**: exploratory testing
**Regression From**: Previous release (1.27.2)

## Steps to Reproduce ##
1. Expand one non-ADLS Gen2 storage account -> Blob Containers.
2. Create a blob container -> Upload one blob.
3. Select the blob then press context menu key.
4. Check whether the context menu shows or not.

## Expected Experience ##
The context menu shows.

## Actual Experience ##
The context menu doesn't show.

**Comment by v-xianya** [comment]
Verified this issue on the build main 20230324.1. Fixed. So, we are going to close it.
